index.html

```diff
@@ -10639,7 +10639,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<pre class="example highlight">&lt;button aria-braillelabel="****"&gt;
   &lt;img alt="4 stars" src="images/stars.jpg"&gt;
 &lt;/button&gt;</pre>
-				<p>In the previous example, a braille display may display "btn ****" in Braille rather than the verbose "btn gra 4 stars".</p>			</div>
+				<p>In the previous example, a braille display can display "btn ****" in Braille rather than the verbose "btn gra 4 stars".</p>			</div>
```

Comment:
Isn't this a normative SHOULD - that is the whole point of braillelabel?
```suggestion
				<p>In the previous example, a braille display SHOULD display "btn ****" in Braille rather than the verbose "btn gra 4 stars".</p>			</div>
```

Reply:
Yes but I thought the spec can't say such normative things about AT.

Reply:
There are 2 AT should statements in the section already
"[Assistive technologies](https://w3c.github.io/aria/#dfn-assistive-technologies) SHOULD use the value of aria-braillelabel when presenting the accessible name of an element in Braille, but SHOULD NOT change other functionality. For example, an assistive technology that provides aural rendering SHOULD use the accessible name."

Reply:
I think all of these shoulds and should nots be removed, honestly -- see: https://github.com/w3c/aria/issues/1892

Reply:
maybe we should say "would" like you recommend below?

Reply:
Thanks for linking #1892 , @spectranaut -- that's what I was thinking of in my original comment (but failed to find quickly).